drivers/scheduler/k8s/specs/test-sv4-svc/pod.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-sv4-dep-svc
+spec:
+  replicas: 3
```

Comment:
should the number of replica be set as a parameter?
for example:
```
{{ if .Replicas }}
replicas: {{ .Replicas }}
{{ else }}
replicas: 3{{ end }}
```

Reply:
That seems like a good idea. I see that some app specs have that already. Do you know how the "replicas" is passed to the job?

Reply:
it takes from custom config yaml file, path is passed using param
CUSTOM_APP_CONFIG